src/relay.rs

```diff
@@ -309,10 +308,10 @@ impl<T: DuplexTransport + 'static> Network<T> {
 
     /// Returns the gas price for the network as a U256
     /// Change the price by setting or unsetting free in the config.toml
-    pub fn get_gas_price(&self) -> U256 {
+    pub fn get_gas_price(&self, gas_price: U256) -> U256 {
```

Comment:
Odd naming here, get_gas_price but it takes gas_price as a parameter (i get what this is ofc), perhaps rename param to `estimate` or something